translations/monero-core_pt-br.ts

```diff
@@ -568,486 +543,482 @@
         <translation>Continuar</translation>
     </message>
 </context>
 <context>
     <name>PasswordDialog</name>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="88"/>
+        <location filename="../components/PasswordDialog.qml" line="86"/>
         <source>Please enter wallet password</source>
-        <translation>Por favor digite a senha da carteira</translation>
+        <translation>Por favor, digite a senha da carteira</translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="88"/>
-        <source>Please enter wallet password for:&lt;br&gt;</source>
-        <translation>Por favor digite a senha da carteira:&lt;br&gt;</translation>
+        <location filename="../components/PasswordDialog.qml" line="86"/>
+        <source>Please enter wallet password for: </source>
+        <translation>Por favor, digite a senha da carteira para: </translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="156"/>
+        <location filename="../components/PasswordDialog.qml" line="150"/>
         <source>Cancel</source>
         <translation>Cancelar</translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="169"/>
+        <location filename="../components/PasswordDialog.qml" line="161"/>
         <source>Continue</source>
         <translation>Continuar</translation>
     </message>
 </context>
 <context>
     <name>PrivacyLevelSmall</name>
     <message>
-        <location filename="../components/PrivacyLevelSmall.qml" line="100"/>
+        <location filename="../components/PrivacyLevelSmall.qml" line="96"/>
         <source>Low</source>
         <translation>Baixa</translation>
     </message>
     <message>
-        <location filename="../components/PrivacyLevelSmall.qml" line="109"/>
+        <location filename="../components/PrivacyLevelSmall.qml" line="105"/>
         <source>Medium</source>
         <translation>Média</translation>
     </message>
     <message>
-        <location filename="../components/PrivacyLevelSmall.qml" line="118"/>
+        <location filename="../components/PrivacyLevelSmall.qml" line="114"/>
         <source>High</source>
         <translation>Alta</translation>
     </message>
 </context>
 <context>
     <name>ProgressBar</name>
     <message>
-        <location filename="../components/ProgressBar.qml" line="41"/>
-        <source>Establishing connection...</source>
-        <translation>Estabelecendo conexão...</translation>
-    </message>
-    <message>
-        <location filename="../components/ProgressBar.qml" line="58"/>
-        <source>Blocks remaining: %1</source>
-        <translation>Blocos restantes: %1</translation>
+        <location filename="../components/ProgressBar.qml" line="38"/>
+        <source>%1 blocks remaining: </source>
+        <translation>%1 blocos restantes: </translation>
```

Comment:
%1 is a confusing variable here - it represents the text Wallet/Node rather than a number of blocks. [See my note for the French review](https://github.com/monero-project/monero-gui/pull/1390#discussion_r187590964).

So the translation should be something like "Blocos restantes para %1: "

Reply:
Good catch, I'll update it.